packages/wallets/src/components/SentEmailContent/SentEmailContent.tsx

```diff
@@ -18,92 +15,38 @@ type SentEmailContentProps = {
 };
 
 const SentEmailContent: FC<SentEmailContentProps> = ({ description, isInvestorPassword = false, platform }) => {
-    const [shouldShowResendEmailReasons, setShouldShowResendEmailReasons] = useState(false);
-    const [hasCountdownStarted, setHasCountdownStarted] = useState(false);
     const { data } = useSettings();
     const { mutate: verifyEmail } = useVerifyEmail();
-    const { isMobile } = useDevice();
     const mt5Platform = PlatformDetails.mt5.platform;
-    const { title } = PlatformDetails[platform ?? mt5Platform];
-    const titleSize = 'md';
-    const descriptionSize = 'sm';
-    const emailLinkSize = isMobile ? 'lg' : 'md';
-    const [count, { resetCountdown, startCountdown }] = useCountdown({
-        countStart: 60,
-        intervalMs: 1000,
-    });
-
-    useEffect(() => {
-        if (count === 0) setHasCountdownStarted(false);
-    }, [count]);
+    const { isMobile } = useDevice();
 
     const { data: activeWallet } = useActiveWalletAccount();
 
     const mt5ResetType = isInvestorPassword
         ? 'trading_platform_investor_password_reset'
         : 'trading_platform_mt5_password_reset';
 
+    useEffect(() => {
+        if (data?.email) {
+            verifyEmail({
+                type: platform === mt5Platform ? mt5ResetType : 'trading_platform_dxtrade_password_reset',
+                url_parameters: {
+                    redirect_to: platformPasswordResetRedirectLink(platform ?? mt5Platform, activeWallet?.is_virtual),
+                },
+                verify_email: data?.email,
+            });
+        }
+    }, [data.email, activeWallet?.is_virtual, mt5Platform, mt5ResetType]);
+
     return (
         <div className='wallets-sent-email-content'>
             <WalletsActionScreen
-                description={description ?? `Please click on the link in the email to change your ${title} password.`}
-                descriptionSize={descriptionSize}
+                description={description ?? `Please click on the link in the email to reset your password.`}
+                descriptionSize={isMobile ? 'lg' : 'sm'}
```

Comment:
<img width="916" alt="Screenshot 2024-04-04 at 1 10 30 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/77470307-0589-4e66-96a1-b0bcf6fb6bd8">

Reply:
For change password in Deriv X:
<img width="1098" alt="Screenshot 2024-04-05 at 11 13 50 AM" src="https://github.com/binary-com/deriv-app/assets/103016120/76e2bb42-0033-4089-9d99-3bf8ce8f40a9">